services/movie_session.py

```diff
@@ -0,0 +1,59 @@
+from django.db.models import QuerySet
+import datetime
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
```

Comment:
The `movie_show_time` parameter should be of type `datetime.datetime`, not `datetime`. Ensure you import `datetime` correctly and use `datetime.datetime` for the type hint.